docs/reference/outlets.md

```diff
@@ -0,0 +1,179 @@
+---
+permalink: /reference/outlets.html
+order: 04
+---
+
+# Outlets
+
+_Outlets_ let you reference Stimulus _controller instances_ and their _controller element_ from within another Stimulus Controller by using CSS selectors.
+
+The use of outlets helps with cross-controller communication and coordination as an alternative to dispatching custom events on controller elements.
```

Comment:
```suggestion
The use of Outlets helps with cross-controller communication and coordination as an alternative to dispatching custom events on controller elements.
```

It appears that Outlet is a proper noun throughout the rest of the page so maybe it should be here also